src/pyfaf/storage/jsontype.py

```diff
@@ -18,5 +18,5 @@ def process_result_value(self, value, dialect):
     def process_literal_param(self, value, dialect):
         return value
 
-    def python_type(self):
+    def python_type(self): # pylint: disable=invalid-overridden-method
```

Comment:
Probably should rather just include a decorator? https://github.com/sqlalchemy/sqlalchemy/blob/8906617cd180f45219de2dd34dff8284f168fa9a/lib/sqlalchemy/sql/type_api.py#L392

Reply:
Done.